index.js

```diff
@@ -230,13 +230,35 @@ module.exports = {
 				'**/?(*.)+(spec|test).[jt]s?(x)',
 				'**/*-specs.{js,jsx,ts,tsx}'
 			],
+			plugins: ['jest', 'testing-library'],
 			excludedFiles: ['tests/screenshot/**/*', 'tests/ui/**/*'],
 			env: {
-				jest: true
+				jest: true,
+				'jest/globals': true,
```

Comment:
Do we need both of these options?

Reply:
Let me check

Reply:
As eslint developer site described, both options are correct(provide jest global variables).
Especially, `jest: true` don't need to add `eslint-plugin-jest` module but `jest/globals: true` need to.
I will remove this.